kayenta-2.46.yaml

```diff
@@ -0,0 +1,83 @@
+package:
+  name: kayenta-2.46
+  version: "2.46.0"
+  epoch: 0
+  description: Automated Canary Service
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-${{vars.java-version}}-jre
+    provides:
+      - kayenta=${{package.full-version}}
+
+vars:
+  java-version: 17 # https://github.com/spinnaker/kayenta/blob/a69573be9d8fededacf80817e4151a2c8df41276/Dockerfile.slim#L3
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - openjdk-${{vars.java-version}}
+      - openjdk-${{vars.java-version}}-default-jvm
+  environment:
+    JAVA_HOME: /usr/lib/jvm/java-${{vars.java-version}}-openjdk
+    GRADLE_OPTS: "-Dorg.gradle.daemon=false -Xmx2g -Xms2g"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/spinnaker/kayenta
+      expected-commit: a69573be9d8fededacf80817e4151a2c8df41276
+      tag: v${{package.version}}
+
+  - uses: auth/gradle
+
+  - uses: patch
+    with:
+      patches: upgrade-dependencies.patch
+
+  - runs: |
+      ./gradlew --info --stacktrace build kayenta-web:installDist \
+          -x :kayenta-core:test \
+          -x :kayenta-integration-tests:test
+
+      mkdir -p ${{targets.contextdir}}/opt/kayenta
```

Comment:
I am not super sure, but should this be a compat package?

Reply:
I noticed that we were copying a lot of other files into /opt, not just the binary. I think it would be better to keep it that way, as it makes managing everything in /opt much simpler just like upstream did, what do you think?